From 88de4fc8b7d4761c32c0ace3c79975671571f50c Mon Sep 17 00:00:00 2001
From: Jan Beich <jbeich@FreeBSD.org>
Date: Sun, 15 Nov 2020 05:36:13 +0000
Subject: [PATCH 1/3] seat: chase swaywm/wlroots@7693f61d8138

seat.c:263:56: error: use of undeclared identifier 'WLR_KEY_PRESSED'
        if ((modifiers & WLR_MODIFIER_ALT) && event->state == WLR_KEY_PRESSED) {
                                                              ^
---
 seat.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/seat.c b/seat.c
index 651ed32..dea384c 100644
--- a/seat.c
+++ b/seat.c
@@ -260,7 +260,7 @@ handle_key_event(struct wlr_input_device *device, struct cg_seat *seat, void *da
 
 	bool handled = false;
 	uint32_t modifiers = wlr_keyboard_get_modifiers(device->keyboard);
-	if ((modifiers & WLR_MODIFIER_ALT) && event->state == WLR_KEY_PRESSED) {
+	if ((modifiers & WLR_MODIFIER_ALT) && event->state == WL_KEYBOARD_KEY_STATE_PRESSED) {
 		/* If Alt is held down and this button was pressed, we
 		 * attempt to process it as a compositor
 		 * keybinding. */
-- 
2.29.2

